resources/assets/js/stories/H2Components/Accordion.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withIntl } from "storybook-addon-intl";
+import Accordion from "../../components/H2Components/Accordion";
+
+const stories = storiesOf("H2 Components/Accordion", module).addDecorator(
+  withIntl,
+);
+
+stories.add(
+  "Accordion",
+  (): React.ReactElement => (
+    <section>
+      <Accordion
+        triggerIconPos="right"
+        data-h2-card="white, round"
+        data-h2-margin="b(bottom, 1)"
```

Comment:
I like how you've set up several differently styled options. I think it would be even more useful if you made the most relevant attributes "knobs", so you can test them out from within storybook. The three accordions you have here could each be their own story, with different default knob values.
(This applies to the button and dialog stories as well.)